js/adapt-assessmentArticleModel.js

```diff
@@ -120,6 +120,12 @@ const AssessmentModel = {
   _onDataReady() {
     // register assessment
     Adapt.assessment.register(this);
+    
+    // sanity check the data
+    const questions = this._getCurrentQuestionComponents();
+    if (this.get('_attemptInProgress') && questions.every(q => q.get('_isSubmitted'))) {
+      this.set('_attemptInProgress', false);
```

Comment:
Add something like:
```js
logging.warn(`Assessment ${this.get('_id') _attemptInProgress corrected as all questions are submitted`);
```

The other situation where this isn't the case is if `this.get('_requireCompletionOf') === -1` as another extension will be adding questions.